synapse/lib/socket.py

```diff
@@ -328,6 +328,7 @@ def __init__(self):
 
         #self._plex_sel = selectors.DefaultSelector()
 
+        self._plex_thr = None
```

Comment:
why set this to none?

Reply:
per discussion - this is done to allow the wake functions to not fail during _init_ time.